src/app/screens/App/screens/Login/index.js

```diff
@@ -25,21 +28,33 @@ class LoginContainer extends Component {
     const email = this.state.email;
     const password = this.state.password;
 
-    const emailError = !EMAIL_REGEX.test(email);
-    const passwordError =
-      password.length < 8 ||
-      password.length > 52 ||
-      !HAS_LETTER_REGEX.test(password) ||
-      !HAS_NUMBER_REGEX.test(password);
-
-    if (emailError || passwordError) {
-      this.setState({ emailError, passwordError });
-    } else {
-      localStorage.setItem('currentUser', email);
-      this.props.history.push('/dashboard');
+    const invalidEmail = !EMAIL_REGEX.test(email);
+    const invalidPassword =
+      password.length < 8 || password.length > 52 || !HAS_NUMBER_REGEX.test(password);
+
+    this.setState({
+      emailError: invalidEmail && EMAIL_ERROR, //set either EMAIL_ERROR or false
+      passwordError: invalidPassword && PASSWORD_ERROR
+    });
+
+    if (!invalidEmail && !invalidPassword) {
+      this.requestAuthentication();
     }
   };
 
+  requestAuthentication = async () => {
+    const email = this.state.email;
+    const password = this.state.password;
+
+    axios
+      .post(SESSION_URL, { email, password })
```

Comment:
Then of configuration, when we want to do a request, we create a `service.js` file into service folder, at the same level of app.

something like 
```
import api from '../config/api';

export const fetchClientsList = (filter, pageNumber) =>
  api.get(`clients?filter=${filter}&page=${pageNumber}`);
```